docs/io-app/view-message-list.puml

```diff
@@ -0,0 +1,54 @@
+@startuml name ViewMessageList-ioApp
+participant MessagesHomeScreen
+participant PagerViewContainer
+participant MessageList
+participant Redux
+participant allPaginatedReducer
+participant HandleReloadAllMessages
+note right of HandleReloadAllMessages
+ saga
+end note
+MessagesHomeScreen -> PagerViewContainer : contains
+PagerViewContainer -> MessageList : contains
+return
+PagerViewContainer-->  MessagesHomeScreen
+  PagerViewContainer ->  Redux  : MessagePagePotFromCategorySelector
+  activate PagerViewContainer
+  Redux -> PagerViewContainer  : Pot.None
+    MessageList -> Redux : messageListForCategorySelector
+    activate MessageList
+    Redux -> MessageList : empty UIMessages[]
+    note left
+ UI logic infers
+ empty state
+end note
+  deactivate MessageList
+  PagerViewContainer -> Redux : dispatch reloadAllMessages.request
+deactivate PagerViewContainer
+Redux -> allPaginatedReducer ++ : passes reloadAllMessages.request
+allPaginatedReducer -> Redux --: return new state
+Redux -> HandleReloadAllMessages ++ :  forward reloadAllMessages.request
+HandleReloadAllMessages  -> BE : call Api
```

Comment:
`BE` was not declared among the participants

Reply:
Added